envi/archs/arm/__init__.py

```diff
@@ -52,8 +50,6 @@ class ThumbModule(envi.ArchitectureModule):
     '''
     This architecture module will *not* shift to ARM mode.  Evar.
     '''
-    #Current ARM version working in - Not fully integrated yet
-    archVersion = ('ARMv7A')
```

Comment:
still waiting on this.  please email me if you dont understand what i' asking for.